lib/crystalball/rspec/runner.rb

```diff
@@ -64,6 +64,24 @@ def check_map(out)
           out.puts 'Maps are outdated!' if prediction_builder.expired_map?
         end
       end
+
+      def run_specs(example_groups)
+        check_examples_limit(example_groups)
+        super
+      end
+
+      def check_examples_limit(example_groups)
+        limit = ENV['CRYSTALBALL_SKIP_EXAMPLES_LIMIT']&.to_i || config['examples_limit']
+        return unless limit&.positive?
```

Comment:
incorrect logic.
```
limit = config['examples_limit'].to_i
return if ENV['CRYSTALBALL_SKIP_EXAMPLES_LIMIT'] || !limit.positive?
```

Reply:
I think naming is confusing here. If it's supposed to be an override for `config['examples_limit']`, it should be `ENV['CRYSTALBALL_EXAMPLES_LIMIT']`. If it should be a boolean to enable the limit or not, it should be `ENV['CRYSTALBALL_ENABLE_EXAMPLES_LIMIT']`. Pavel's review assumes it's the latter; for me it's fine either way, but take your pick because it's confusing right now.